src/components/GoalsComponents/DisplayChangesModal/ShowChanges.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { GoalItem } from "@src/models/GoalItem";
+import "./ShowChanges.scss";
+import { InfoCircleOutlined } from "@ant-design/icons";
+
+export const getMovedSubgoalsList = (
+  goalUnderReview: GoalItem | undefined,
+  oldParentTitle: string,
+  newParentTitle: string,
+) => {
+  if (!goalUnderReview) return null;
+
+  return (
+    <div className="move-info-container">
+      <div className="move-info-item">
+        <span className="move-info-label">Goal Being Moved</span>
+        <div className="move-info-value highlight-box">{goalUnderReview.title}</div>
+      </div>
+
+      <div className="move-direction-container">
+        <div className="move-info-item">
+          <span className="move-info-label">From</span>
+          <div className="move-info-value">{oldParentTitle}</div>
+        </div>
+
+        <div className="arrow">→</div>
+
+        <div className="move-info-item">
+          <span className="move-info-label">To</span>
+          <div className="move-info-value">{newParentTitle}</div>
+        </div>
+      </div>
+
+      {newParentTitle === "Non-shared goal" && (
+        <div className="warning-message">
+          <InfoCircleOutlined />
+          <span>The new parent goal is not shared. Changes will be ignored if accepted.</span>
```

Comment:
In my opinion, sharing should be per goal - independant of what the parent goal is.  

If I/you change the parent goal of our shared goal - that should not affect propagation of changes on our shared goal, right?

Reply:
this is for the situation when child is moved out of the shared goal. the above code is stale. 

> If I/you change the parent goal of our shared goal - that should not affect propagation of changes on our shared goal, right?

this is handled differently. if we move the root shared alltogether then the suggestion will discarded altogether by the reciever.

Reply:
Yeah, what I'm saying is that maybe we should be dropping the whole concept of letting shared status depend on some 'root-goal'?  

In my view, when you share a goal - all descendants are also shared (or not with opt-out).  
From then on out - they are 'autonomous' from each other.  

Of course if you delete a tree - all descendants will also be suggested to be deleted - but not the ones you/partner have moved out of the tree previously.  


Basically, this makes sharing/deleting a tree the equivalent of individually sharing/deleting all goals in the tree. The advantage is that after that you are free to move things around individually without any extra complexity/dependency on other goals.

Reply:
I agree that dropping root-goal dependency and treating each goal independently would make the system more flexible and intuitive.

I'll start implementing these changes.